tests/simulate/test_hoomd.py

```diff
@@ -127,6 +128,16 @@ def test_initialize_from_gsd_file(self):
         h = relentless.simulate.HOOMD(op)
         h.run(pot, self.directory)
 
+        # Run in a different directory
+        with self.directory:
+            d = self.directory.directory(
+                "run", create=relentless.mpi.world.rank_is_root
+            )
+            relentless.mpi.world.barrier()
+            op = relentless.simulate.InitializeFromFile(pathlib.Path(f).name)
+            h = relentless.simulate.HOOMD(op)
```

Comment:
You might want to do this to test the setter:
```suggestion
            op.filename = pathlib.Path(f).name
```
Alternatively, you don't need a new HOOMD instance, just do:
```suggestion
            h.initializer = relentless.simulate.InitializeFromFile(pathlib.Path(f).name)
```
Maybe you actually want to do both here?